models/payment/src/payment/payment-draft/presets/sample-data-goodstore/sebastian-mueller-payment.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  LocalizedString,
+  KeyReference,
+  Money,
+} from '@commercetools-test-data/commons';
+import {
+  CustomerDraft,
+  type TCustomerDraft,
+} from '@commercetools-test-data/customer';
+import * as PaymentDraft from '../..';
+import { PaymentMethodInfoDraft } from '../../../../payment-method-info';
+import { PaymentStatusDraft } from '../../../../payment-status';
+import { TransactionDraft, constants } from '../../../../transaction';
+import { TPaymentDraftBuilder } from '../../../types';
+
+const customerSebastianMueller = CustomerDraft.presets.sampleDataGoodStore
+  .sebastianMuller()
+  .build<TCustomerDraft>();
+
+const sebastianMuellerPayment = (): TPaymentDraftBuilder =>
+  PaymentDraft.presets
+    .empty()
+    .customer(
+      KeyReference.presets.customer().key(customerSebastianMueller.key!)
+    )
+    .amountPlanned(Money.random().centAmount(395890).currencyCode('EUR'))
+    .paymentMethodInfo(
+      PaymentMethodInfoDraft.presets
+        .empty()
+        .method('Credit Card')
+        .name(LocalizedString.presets.empty()['de']('Credit Card'))
```

Comment:
I made an assumption that we'd want to use German here - thoughts?

Reply:
I think we should stick with `en-US` and `en-GB` if it's in English.